### DISCUSSION TITLE
Anki deck for lesson 0

### DISCUSSION THREAD

**Discussion by nomasprime** [SethClydesdale/genki-study-resources#208] [Questions]
Just started learning and found this. Thanks for such an excellent resource.

Wondering why there's no deck for lesson 0 though?

**Comment by SethClydesdale** [comment]
Hello! I'm happy you find it helpful! :)

Originally the script we used for scraping the vocab data from the site targeted lessons 1+ since they were the only lessons with dedicated vocabulary exercises at the time. I may have to look into modifying it to also include lesson 0 when I get the chance.

**Reply by fligtar** [reply]
Just wanted to +1 this request as well. Some of the longer greetings from Lesson 0 need a lot of practice, and I think a lot of us would find a Lesson 0 Anki deck really valuable. In the meantime, I found this alternative: https://ankiweb.net/shared/info/2120327137

**Comment by SethClydesdale** [comment]
About a year and a half late, but Lesson 0 is now available on the [deck page](https://sethclydesdale.github.io/genki-study-resources/help/anki-decks/)!
